src/styles/sidebar/components/icon-button.scss

```diff
@@ -1,7 +1,7 @@
 @use "../../mixins/buttons";
 @use "../../variables" as var;
 
-.annotation-action-button {
+.icon-button {
   @include buttons.button-base;
 
   &.is-active {
```

Comment:
Wondering if base-button should get a default `is-active` styling? like grey-6 or something